ext/ffi/ffi_parser.c

```diff
@@ -2166,6 +2166,7 @@ static int parse_declaration_specifiers(int sym, zend_ffi_dcl *dcl) {
 			case YY_ENUM:
 			case YY_ID:
 				sym = parse_type_specifier(sym, dcl);
+				if (((dcl->flags & (ZEND_FFI_DCL_ENUM | ZEND_FFI_DCL_STORAGE_CLASS)) == ZEND_FFI_DCL_ENUM)) zend_ffi_cleanup_dcl(dcl);
```

Comment:
This seems not enough. The following code starts to fail.

```php
<?php
$ffi = FFI::cdef("
    enum {
        TEST_ONE=1,
        TEST_TWO=2,
    } x;
");
?>
```

Uncaught FFI\ParserException: Unsupported type specifier combination at line 5